data/repos.yml

```diff
@@ -513,6 +660,17 @@
   sentry_url: false
   dashboard_url: false
 
+# Assume this should be added to Data Products' portfolio, based on `govuk-knowledge-graph`
+- repo_name: govuk-knowledge-graph-gcp
+  private_repo: true # well not really, but it's "internal"
+  team: "#data-products"
+  type: Data science
+
+# ditto
+- repo_name: govuk-knowledge-graph-search
+  team: "#data-products"
+  type: Data science
```

Comment:
@nacnudus

Reply:
@maxf Should the govuk tag be removed, or should we document this alongside GOV.UK apps?

Reply:
Sounds good 👍 Can I ask you to raise a PR adding this repo to the dev docs, @nacnudus / @maxf ?